plugins/turing/manager/experiment_manager.go

```diff
@@ -142,37 +141,20 @@ func (em *experimentManager) MakeTreatmentServicePluginConfig(
 	projectID int,
 ) (*config.Config, error) {
 	pluginConfig := &config.Config{
-		Port:                    em.TreatmentServicePluginConfig.Port,
-		ProjectIds:              []string{strconv.Itoa(projectID)},
-		AssignedTreatmentLogger: em.TreatmentServicePluginConfig.AssignedTreatmentLogger,
-		DebugConfig:             em.TreatmentServicePluginConfig.DebugConfig,
-		DeploymentConfig:        em.TreatmentServicePluginConfig.DeploymentConfig,
-		ManagementService:       em.TreatmentServicePluginConfig.ManagementService,
-		MonitoringConfig:        em.TreatmentServicePluginConfig.MonitoringConfig,
-		SwaggerConfig:           em.TreatmentServicePluginConfig.SwaggerConfig,
-		NewRelicConfig:          em.TreatmentServicePluginConfig.NewRelicConfig,
-		SentryConfig:            em.TreatmentServicePluginConfig.SentryConfig,
-		SegmenterConfig:         *treatmentServiceConfig.SegmenterConfig,
-	}
-	messageQueueKind := *treatmentServiceConfig.MessageQueueConfig.Kind
-	switch messageQueueKind {
-	case schema.MessageQueueKindPubsub:
-		pluginConfig.MessageQueueConfig = common_mq_config.MessageQueueConfig{
-			Kind: "pubsub",
-			PubSubConfig: &common_mq_config.PubSubConfig{
-				Project:              *treatmentServiceConfig.MessageQueueConfig.PubSub.Project,
-				TopicName:            *treatmentServiceConfig.MessageQueueConfig.PubSub.TopicName,
-				PubSubTimeoutSeconds: em.TreatmentServicePluginConfig.PubSubTimeoutSeconds,
-			},
-		}
-	case schema.MessageQueueKindNoop:
-		pluginConfig.MessageQueueConfig = common_mq_config.MessageQueueConfig{
-			Kind: "",
-		}
-	default:
-		return nil, fmt.Errorf("invalid message queue kind (%s) was provided", messageQueueKind)
+		Port:                          em.TreatmentServicePluginConfig.Port,
+		ProjectIds:                    []string{strconv.Itoa(projectID)},
+		AssignedTreatmentLogger:       em.TreatmentServicePluginConfig.AssignedTreatmentLogger,
+		DebugConfig:                   em.TreatmentServicePluginConfig.DebugConfig,
+		DeploymentConfig:              em.TreatmentServicePluginConfig.DeploymentConfig,
+		MessageQueueConfig:            em.TreatmentServicePluginConfig.MessageQueueConfig,
```

Comment:
Inclusion of the `MessageQueueConfig` here means that this config now depends explicitly on whatever is configured on the Turing API server (this config gets passed to the XP plugin whenever a Turing router is deployed).